catalog/core/catalog-core-solr/src/main/java/ddf/catalog/source/solr/SolrFilterDelegate.java

```diff
@@ -187,9 +185,12 @@ private void combineXpathFilterQueries(SolrQuery query, List<SolrQuery> subQueri
             for (String xpath : xpathIndexes) {
                 indexes.add("(" + XPATH_FILTER_QUERY_INDEX + ":\"" + xpath + "\")");
             }
-            String index = XPATH_QUERY_PARSER_PREFIX + StringUtils.join(indexes, operator);
```

Comment:
This is related to XPath pre-filtering being disabled.  I think I will go back and add a comment about DDF-1882 here as well.